### PR TITLE
Remove legacy Iris compatibility

### PR DESCRIPTION
**Pull request by qualk**
Removes legacy Iris compatibility, adds hard failure (should it be a soft failure?)

### CODE REVIEW

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -56,6 +56,9 @@
 		"porting_lib_entity": ">=2.3.4+1.20.1",
 		"mkb": ">=1.2.0"
 	},
+	"breaks": {
```
Comment (qualk) [inline src/main/resources/fabric.mod.json:59]: Should it be a soft failure (`"conflicts"`)?

**Comment by qualk** [comment]
In commit [3587e86](https://github.com/We-Cant-Coding/TACZ-Fabric/commit/3587e863b5c81bcdb92a06f079c0b2c8f55cdb09), `gradlew` was made non-executable (755 to 644), resulting in the GitHub actions failing. Can I resolve that in this PR, or should I make another PR?

**Comment by LaeDev** [comment]
> In commit [3587e86](https://github.com/We-Cant-Coding/TACZ-Fabric/commit/3587e863b5c81bcdb92a06f079c0b2c8f55cdb09), `gradlew` was made non-executable (755 to 644), resulting in the GitHub actions failing. Can I resolve that in this PR, or should I make another PR?

Do it in other PR :)

**Comment by qualk** [comment]
> > In commit [3587e86](https://github.com/We-Cant-Coding/TACZ-Fabric/commit/3587e863b5c81bcdb92a06f079c0b2c8f55cdb09), `gradlew` was made non-executable (755 to 644), resulting in the GitHub actions failing. Can I resolve that in this PR, or should I make another PR?
> 
> Do it in other PR :)

#23 :)

**Review by LaeDev — APPROVED** [review]

**Comment by LaeDev** [comment]
i didnt checked the code lmao is it able to build?

**Comment by qualk** [comment]
Doesn't seem like it, even though it built for me. I think it's because the Modrinth maven is bit weird at times. Can you revert the PR?

**Comment by LaeDev** [comment]
Sure. Hold on a sec

**Pull request merged**